src/Reflex/Class.hs

```diff
@@ -911,6 +911,14 @@ zipDynWith f da db =
         return $ Just $ f a b
   in unsafeBuildDynamic (f <$> sample (current da) <*> sample (current db)) ec
 
+instance (Reflex t, Semigroup a) => Semigroup (Dynamic t a) where
+  (<>) = zipDynWith (<>)
```

Comment:
This is `liftA2` no?

Reply:
I believe `liftA2` only recently became a method of `Applicative`.  For versions where it's as method, `liftA2` and `zipDynWith` are equivalent; for older versions, `zipDynWith` is more performant.

Reply:
should be, yes.

Reply:
@ryantrinkle Makes sense. Thanks.